client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -234,6 +236,15 @@ export default function getThankYouPageUrl( {
 		);
 	}
 
+	// Asismet site-less checkout
+	if ( isAkismetSitelessCheckout ) {
+		// extract a product from the cart, in siteless checkout there should only be one
+		const productSlug = cart?.products[ 0 ]?.product_slug ?? 'no_product';
```

Comment:
We could potentially add a check to ensure that this `productSlug` is an Akismet product, do you think that would be useful, or unnecessary?

Reply:
Likely redundant here - we can handle validation like this within the cart itself :)